.storybook/webpack.config.ts

```diff
@@ -1,10 +1,19 @@
 import webpack from "webpack"
 
 const path = require("path")
-const TsconfigPathsPlugin = require("tsconfig-paths-webpack-plugin")
 const MiniCssExtractPlugin = require("mini-css-extract-plugin")
 
 const configAdjuster = ({ config }: { config: webpack.Configuration }) => {
+    // baseDir is necessary to make webpack.config.ts use the correct path both in TS as well as in
+    // transpiled JS form
+    let baseDir = path.resolve(__dirname, "..")
+    baseDir =
+        path.basename(baseDir) === "itsJustJavascript"
+            ? path.resolve(baseDir, "..")
+            : baseDir
```

Comment:
There's _still_ some funky path stuff going on here, I'm sorry :/

Reply:
Hey at least it's contained to storybook and webpack! :)

nit: I wonder if we can remove the leading "." in the storybook folder since you are passing the dir as a param now in the command line argument. Not sure if storybook hards code for that directory or not.

Reply:
I don't think that's hardcoded anywhere, but I kind of like it?
This way we have folders starting with `.` for tooling config like Storybook and VS Code, and everything else is actual code.